<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1439</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>17</y>
     <width>120</width>
     <height>227</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>500</height>
    </size>
   </property>
   <property name="title">
    <string>Input</string>
   </property>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>69</width>
      <height>48</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QRadioButton" name="radioButton_1">
     <property name="text">
      <string>Camera</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="text">
      <string>File</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>120</width>
     <height>226</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>500</height>
    </size>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>82</width>
      <height>96</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QCheckBox" name="checkBox_1">
     <property name="text">
      <string>landmarks</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>BBoxs</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="text">
      <string>7 Emotions</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="text">
      <string>VA Value</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="image_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
